addons/binding/org.openhab.binding.lgwebos/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="lgwebos" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+	<name>LG webOS Binding</name>
+	<description>Binding to connect LG's WebOS based smart TVs based on Connect SDK</description>
+	<author>Sebastian Prehn</author>
+
+
+	<config-description>
```

Comment:
This configuration should not be necessary, see my comment below on how to retrieve the local IP.

Reply:
Done